tasks/php_install.yml

```diff
@@ -1,4 +1,23 @@
 ---
+- name: "[DEBIAN REPO] Debian - repository for php"
+  when: (ansible_facts['distribution'] == "Debian")
+  block:
+    - name: Debian php repository - install required packages
+      ansible.builtin.package:
+        name: gpg
+        state: present
+    - name: Debian php repository - GPG key
+      ansible.builtin.apt_key:
+        url: https://packages.sury.org/php/apt.gpg
+        state: present
+    - name: Debian php repository - APT source
+      ansible.builtin.apt_repository:
+        repo: deb https://packages.sury.org/php/ {{ ansible_distribution_release }} main
+        state: present
+    - name: Debian php repository - APT cache update
+      ansible.builtin.apt:
+        update_cache: true
```

Comment:
by default the apt-repository module update the cache when a change is made so I'm not sure this task is required 
(https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_repository_module.html#parameter-update_cache)

Reply:
Yeah, you've got right. I've added that before, because molecule installed 7.4 even after adding repo. But I've forgotten to change default version to `8.1` in defaults.